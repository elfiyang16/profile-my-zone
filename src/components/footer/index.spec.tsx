import React from "react"
import { shallow } from "enzyme"
import Footer from "./"

describe("<Footer />", () => {
  let wrapper: any
  beforeEach(() => {
    wrapper = shallow(<Footer />)
  })

  it("should render", () => {
    expect(wrapper).toMatchSnapshot()
  })

  it("should render footer ", () => {
    expect(wrapper.find(".site-footer").length).toEqual(1)
  })
})
