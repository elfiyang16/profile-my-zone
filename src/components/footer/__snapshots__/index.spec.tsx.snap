// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render 1`] = `
<footer
  className="site-footer"
>
  Â© 
  2021
  , built by Elfi with Gatsby under
   
  <a
    href="https://opensource.org/licenses/MIT"
  >
    MIT License
  </a>
</footer>
`;
